README.md

```diff
@@ -133,6 +134,31 @@ python analyze.py --config_file config.ini --log_level DEBUG
 
 See [Generic Command Line Parameters](#generic-command-line-parameters) for more details.
 
+# Analysis with sclite
+`sclite` is an open source tool designed to evaluate STT transcription results. More information about `sclite` can be found here -- https://people.csail.mit.edu/joe/sctk-1.2/doc/sclite.htm#sclite_name_0.
+
+This repo provides a wrapper script `analyze_with_sclite.py`, which can be used to run `sclite` and format its output.
+
+## Setup
+1. `reference_transcriptions_file` and `stt_transcriptions_file` must be populated in `config.ini` and exist on the filesystem.
+1. `sclite_directory` must be uncommented and populated with the directory that hold the `sclite` executable
+    1. To install `sclite` follow the instructions here -- https://github.com/usnistgov/SCTK#sctk-basic-installation
+
+## Execution
+
+```
+python analyze_with_sclite.py --config config.ini --log_level INFO
+```
+See [Generic Command Line Parameters](#generic-command-line-parameters) for more details.
+
+## Results
```

Comment:
I think some of these are results (`sclite_wer_summary.json`) and some are "just" intermediate output.  I would separate out the files that user attention should be drawn towards, give them more prominent placement.

Reply:
Separated out the two input files `ctm` and `stm` from the actual output files and rearranged the output files to reflect increasing level of detail.